plugins/modules/ipasudorule.py

```diff
@@ -88,9 +88,26 @@
     required: false
     type: list
   cmdcategory:
```

Comment:
DOCUMENTATION still contains options cmd and cmdgroup. These should not be there anymore. EXAMPLES also need to be adapted.